src/widgets.js

```diff
@@ -350,8 +350,13 @@ define([
             super_updateValue();
         };
 
+        widget._setURLHash = _.debounce(function(mug){
+                mug.form.vellum._setURLHash(mug)
+            }, 500);
```

Comment:
Nit: non-standard indentation
```suggestion
            mug.form.vellum._setURLHash(mug)
        }, 500);
```